packages/blocks/src/class-blocks.php

```diff
@@ -17,6 +17,144 @@
  * @since 9.0.0
  */
 class Blocks {
+	/**
+	 * Wrapper function to safely register a gutenberg block type
+	 *
+	 * @param string $slug Slug of the block.
+	 * @param array  $args Arguments that are passed into register_block_type.
+	 *
+	 * @see register_block_type
+	 *
+	 * @since 6.7.0
+	 *
+	 * @return WP_Block_Type|false The registered block type on success, or false on failure.
+	 */
+	public static function jetpack_register_block( $slug, $args = array() ) {
+		if ( 0 !== strpos( $slug, 'jetpack/' ) && ! strpos( $slug, '/' ) ) {
+			_doing_it_wrong( 'jetpack_register_block', 'Prefix the block with jetpack/ ', '7.1.0' );
```

Comment:
Should this be "Jetpack 7.1.0" since it is in a package that may not be part of Jetpack the plugin anymore?

Reply:
👍 98552cc73357864e74254af663228c65eeb0e35e